benefits/eligibility/forms.py

```diff
@@ -53,32 +53,74 @@ class EligibilityVerificationForm(forms.Form):
         "missing": _("This field is required."),
     }
 
-    def __init__(self, verifier: models.EligibilityVerifier, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(
+        self,
+        title,
+        headline,
+        blurb,
+        name_label,
+        name_placeholder,
+        name_help_text,
+        sub_label,
+        sub_placeholder,
+        sub_help_text,
+        name_max_length=None,
+        sub_input_mode=None,
+        sub_max_length=None,
+        sub_pattern=None,
```

Comment:
These comments from the data file could become docstring or comments on this file to describe these:

```python
    # A regular expression used to validate the 'sub' API field before sending to this verifier
    form_sub_pattern = models.TextField(null=True)
    # Input mode can be "numeric", "tel", "search", etc. to override default "text" keyboard on mobile devices
    form_input_mode = models.TextField(null=True)
    # The maximum length accepted for the 'sub' API field before sending to this verifier
    form_max_length = models.PositiveSmallIntegerField(null=True)
    # The maximum length accepted for the 'name' API field before sending to this verifier
    form_name_max_length = models.PositiveSmallIntegerField(null=True)
```

Reply:
Thanks for pasting here 🙏 I'll add those.

Reply:
[dae5371](https://github.com/cal-itp/benefits/pull/1657/commits/dae53710d5769c7248f4d050c4c2682fcbca5631)

Reply:
🎉🎉🎉

Reply:
Ahh, typo... one more rebase incoming